defs/panda.def

```diff
@@ -168,6 +134,55 @@ void setXYZE(double px, double py, double pz, double e) override
   mass_ = std::sqrt(e * e - p * p);
 }
 
+% Note that SecondaryVertex stuff is copy-pasted from Vertex and RecoVertex
+% Multiple inheritance isn't working due to inheritance from Element
+[SecondaryVertex>ParticleM]
+x/F
+y/F
+z/F
+TVector3 position() const { return TVector3(x, y, z); }
+ntrk/s
+ndof/F
+chi2/F
+double normChi2() const { return ndof != 0. ? chi2 / ndof : chi2 * 1.e+6; }
+% pfRangeMax/s ??
```

Comment:
You can remove this line. Having pfRangeMax on the Vertex object is the gimmick that allows us to not save the associated vertex index for every single PF candidate - vertices instead specify the ranges of candidates that are associated with them.